src/features/advanced_analytics_panel/atoms/advancedAnalyticsResource.ts

```diff
@@ -22,7 +22,7 @@ export const advancedAnalyticsResourceAtom = createResourceAtom(
             errorsConfig: { dontShowErrors: true },
           },
         );
-      } catch (e) {
+      } catch (e: any) {
```

Comment:
Why any here?

Reply:
to avoid ts error for type `unknown`
added a function to fix it